### ISSUE TITLE
[Bug]: PydanticDeprecatedSince20: The `dict` method is deprecated

### ISSUE THREAD

**Issue by jamesbraza** [BerriAI/litellm#5987]
### What happened?

https://github.com/BerriAI/litellm/blob/v1.48.7/litellm/utils.py#L8348 calls `.dict()`, which leads to a `PydanticDeprecatedSince20`

### Relevant log output

```shell
/home/runner/work/repo/repo/.venv/lib/python3.11/site-packages/pydantic/main.py:1114: PydanticDeprecatedSince20: The `dict` method is deprecated; use `model_dump` instead. Deprecated in Pydantic V2.0 to be removed in V3.0. See Pydantic V2 Migration Guide at https://errors.pydantic.dev/2.9/migration/
    warnings.warn('The `dict` method is deprecated; use `model_dump` instead.', category=PydanticDeprecatedSince20)
```

**Comment by githubnewsettler** [comment]
How to deal with the problem?

**Comment by toutantic** [comment]
Seems this issue have been fixed in this part of litellm but have been re-introduce somewhere else

My logs are full of this error with ltellm version 1.63.14:

`back-1    | /env/.venv/lib/python3.11/site-packages/litellm/litellm_core_utils/streaming_handler.py:1736: PydanticDeprecatedSince20: The `dict` method is deprecated; use `model_dump` instead. Deprecated in Pydantic V2.0 to be removed in V3.0. See Pydantic V2 Migration Guide at https://errors.pydantic.dev/2.10/migration/`

**Comment by jamesbraza** [comment]
Yeah there's a `dict` call here now: https://github.com/BerriAI/litellm/blob/v1.63.14-stable/litellm/litellm_core_utils/streaming_handler.py#L1736

**Comment by jamesbraza** [comment]
Not stale, the warning is still emitted

**Comment by mickaelandrieu** [comment]
its not stale at all, the only people that dont react to the issue are the ones in charge of the maintenance of the project
